ProffieOS.ino

```diff
@@ -705,6 +706,12 @@ int prop_GetBulletCount() {
 }
 #endif
 
+#ifdef PROP_HAS_GETBLASTERMODE
```

Comment:
This should be defined in props/blaster.h

Reply:
oops. Edit was sitting there, I didn't hit go.
Done.